resources/services/lambda/functions.go

```diff
@@ -1423,6 +1423,10 @@ func resolveFunctionEventSourceMappingsSourceAccessConfigurations(ctx context.Co
 	return diag.WrapError(resource.Set(c.Name, data))
 }
 
-// ====================================================================================================================
-//                                                  User Defined Helpers
-// ====================================================================================================================
+func resolveFunctionsTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	r := resource.Item.(*lambda.GetFunctionOutput)
+	if r.Tags == nil {
+		return resource.Set(c.Name, make(map[string]string))
+	}
+	return resource.Set(c.Name, r.Tags)
```

Comment:
Since `resource.Set(c.Name, r.Tags)` returns an error, we should wrap it with `diag.WrapError()`.
This is configured in https://github.com/cloudquery/.github/blob/83792feb7d99ac5ff02017a20626bf8480335a3c/misc/providers/.golangci.yml#L32